layouts/partials/language-switcher.html

```diff
@@ -1,5 +1,16 @@
-<ul class="language-select">
-{{ range $.Site.Home.AllTranslations }}
-<li><a href="{{ .Permalink }}">{{ .Language.LanguageName }}</a></li>
+{{ if .IsTranslated }}
+    {{ $pageLang := .Page.Lang}}
```

Comment:
I tried to get rid of this intermittent variable (`$pageLang`), but it doesn't work without it.
So I decided to leave it in this way.

Reply:
I think, it's correct, because `.Page` is different for each translation